torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10672,19 +10672,16 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
            aten_name='linalg_qr',
            op=torch.linalg.qr,
            dtypes=floating_and_complex_types(),
-           # batched gradients do not work for empty inputs
-           # https://github.com/pytorch/pytorch/issues/50743#issuecomment-767376085
-           check_batched_gradgrad=False,
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
+           # In-place ops
+           check_batched_gradgrad=False,
+           sample_inputs_func=sample_inputs_linalg_qr_geqrf,
            skips=(
-               DecorateInfo(unittest.expectedFailure, 'TestCompositeCompliance', 'test_backward'),
+               # The test is wrong https://github.com/pytorch/pytorch/pull/76115/files#r854455360
                DecorateInfo(unittest.expectedFailure, 'TestCompositeCompliance', 'test_forward_ad'),
```

Comment:
Could you please clarify? What is wrong exactly?

Reply:
They don't handle strides properly in the test for empty tensors.

Reply:
See the comment linked there for further discussion.

Reply:
Sorry, I am missing something, but the link is to the PRs hash. Where do I find the comment?

Reply:
Or do you mean the link that was removed in your PR regarding empty inputs?